packages/central-server/src/apiV2/syncGroups/DeleteSyncGroups.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { DeleteHandler } from '../DeleteHandler';
+import { assertAnyPermissions, assertBESAdminAccess } from '../../permissions';
+import { assertSyncGroupEditPermissions } from './assertSyncGroupPermissions';
+
+export class DeleteSyncGroups extends DeleteHandler {
+  async assertUserHasAccess() {
+    const syncGroupChecker = accessPolicy =>
+      assertSyncGroupEditPermissions(accessPolicy, this.models, this.recordId);
+
+    await this.assertPermissions(assertAnyPermissions([assertBESAdminAccess, syncGroupChecker]));
```

Comment:
We may need to consider how to change this permission check... I suspect there will be some lesmis users who want to configure the sync group but don't have BES Admin...